src/index.ts

```diff
@@ -146,22 +177,11 @@ export default class OASNormalize {
               return schema;
             }
 
-            return converter.convertObj(schema, { anchors: true }).then((options: { openapi: OpenAPI.Document }) => {
-              return options.openapi;
-            });
+            return converter
+              .convertObj(schema, { anchors: true })
+              .then((options: { openapi: OpenAPI.Document }) => options.openapi);
           })
           .catch(err => Promise.reject(err));
-      }
-
-      return Promise.reject(new Error('The supplied API definition is unsupported.'));
-    });
-  }
-
-  /**
-   * Retrieve the OpenAPI or Swagger version of the current API definition.
-   *
-   */
-  version() {
```

Comment:
So we actually need a function like this in `rdme`: for our spec selection process, [we use this function](https://github.com/readmeio/rdme/blob/e264f6881a6026a7b540d5e72dedb36c11964cb9/src/lib/prepareOas.ts#L67) to detect potential API definitions prior to any validation/other processing we do. Could we export a `isApiDefinition` boolean or something that's just returning something along the lines of `isOpenAPI || isPostman || isSwagger`?

Reply:
Ok I've resurrected this a bit with `isAPIDefiniton` and `getAPIDefinitonType` methods that should be able to replace `.version`.